test/ts-use/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "ts-use",
+  "private": true,
+  "dependencies": {
+    "multiformats": "file:../.."
```

Comment:
this needs to be `../../dist` to properly test the distributable files, but then an `npm install` needs to be run from within there

i.e. this isn't actually testing what we distribute and is not properly exercising the faulty `typesVersions` field

Reply:
No this needs to point to the package root, which then has `typesVersion` field that tells TS to look into `dist`. It is true that it will not exactly catch problems of something not getting published into npm, but it would still catch problems e.g. if `typesVersion` isn't pointing to right directory or if TS emitted files have problems (like some that we have discovered in js-ipfs).

Reply:
The "package root" is the dist/ directory, the root of this repo doesn't get published to npm, the publish procedure goes into dist/ and does an `npm publish` from there.

```
"publish": "npm_config_yes=true npx ipjs@latest publish"
```

`ipjs publish` does a build to dist/ and publish _from_ dist/

in its current form, this package's typescript definitions can't be consumed by dependents. See #49 for fix.